tests/netconf/netconf_protocol_test.py

```diff
@@ -3,15 +3,20 @@
 import sys
 import unittest
 
-from fake_switches.netconf import RUNNING, dict_2_etree
-from fake_switches.netconf.capabilities import filter_content
-from fake_switches.netconf.netconf_protocol import NetconfProtocol
 from hamcrest import assert_that, ends_with, equal_to, has_length, has_key
 from hamcrest.core.base_matcher import BaseMatcher
+from lxml import etree
 from lxml.etree import _Element
 from mock import Mock
+from ncclient import xml_
 from ncclient.xml_ import to_ele, to_xml
 
+from fake_switches.netconf import RUNNING, dict_2_etree
+from fake_switches.netconf.capabilities import filter_content
+from fake_switches.netconf.netconf_protocol import NetconfProtocol
+
+xml_.parser = etree.XMLParser(recover=True)
```

Comment:
what's this for?

Reply:
Makes XML less picky...
Like, for example, having the namespace defined on the 2nd line of a packet would normally break. That being said, I think it makes the tests easier to read.

Reply:
OH it's the test, nvm
* runs and hide

Reply:
Now that I think about it, I am not sure I want the tests to be "less picky"...
Will remove and fix all the XMLs